extra/gitdm/company-map

```diff
@@ -44,3 +44,8 @@ sentinel.tech Sentinel.tech
 timesys.com Timesys Corporation
 tiscali.it Tiscali S.p.a.
 jester@theyoctojester.info Northern.tech
+j.bennett@engineeredarts.co.uk Engineered Arts
+code@jackben.net Engineered Arts
+sven.schwermer@disruptive-technologies.com Disruptive Technologies
+tobias.zimmerer@zf.com ZF
+r.schwarz@sotec.eu SOTEC
```

Comment:
For addresses where the domain clearly belongs to the company, it is better to add them without a username, so that future contributors are also covered. This is in contrast to gmail addresses, which need to be specified per user. You can see `tiscali.it` above is done this way.